### ISSUE TITLE
Horizontal change of resolution

### ISSUE THREAD

**Issue by guillaumevernieres** [JCSDA-internal/soca#339]
## Description
Currently, "change_resolution" in State and Increments are not implemented. As a first step, implement methods to interpolate a State or an Increment between 2 geometries which have the same vertical coordinate. 

## Solution
- Simple/naive 2D interpolation (by @guillaumevernieres or @travissluka ) that will be deemed sufficient for our DA needs
- More elaborate case that will enable the interpolation of the entire model state (This work will be done by @clouden90 and probably requires its own issue)

I both cases, we will investigate the use of the available fms interpolation package and the unstructured one in oops.

## Dependencies
- This work cannot start before #210 is resolved

**Comment by guillaumevernieres** [comment]
@clouden90 , assign yourself or @flampouris , please assign @clouden90 . We'll talk today (05/07/2020) about @clouden90 's level of involvement for this step.

**Comment by clouden90** [comment]
> @clouden90 , assign yourself or @flampouris , please assign @clouden90 . We'll talk today (05/07/2020) about @clouden90 's level of involvement for this step.

@guillaumevernieres , sounds perfect. Let's talk about the plans, step by step for #339 today.
